webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/ApiDatastructures.scala

```diff
@@ -259,6 +259,7 @@ trait StartsAtVersion13 extends EndpointSchema { val versions = ApiV.From(13) }
 trait StartsAtVersion14 extends EndpointSchema { val versions = ApiV.From(14) }
 trait StartsAtVersion15 extends EndpointSchema { val versions = ApiV.From(15) }
 trait StartsAtVersion16 extends EndpointSchema { val versions = ApiV.From(16) }
+trait StartsAtVersion17 extends EndpointSchema { val versions = ApiV.From(17) }
```

Comment:
this also need to be changed in the API doc

Reply:
there is no API doc for the moment, problem solved :)

Reply:
wrong version issue reported by @ncharles [here](https://github.com/Normation/rudder/pull/4626#discussion_r1112837815)

Reply:
yes but you need to change version to 17 in the doc too